packages/dioxus-blitz/src/window.rs

```diff
@@ -308,6 +398,47 @@ impl<'a, Doc: DocumentLike> View<'a, Doc> {
         self.waker = None;
         self.renderer.suspend();
     }
+
+    #[cfg(feature = "accesskit")]
+    fn build_accessibility_tree(&mut self) {
```

Comment:
Can we move this function (and probably `build_node` above too) into their own `accesskit` module?